src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Interop/IVsAppId.cs

```diff
@@ -0,0 +1,289 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
Do we have do this?  There isn't a PIA available?

Reply:
Yep. I have thread with the shell team on this and this was the recommendation.

Reply:
Note that we could remove all the enums and just leave the one constant we are using.  It would certainly reduce the size of the file.

Reply:
The size of the file doesn't bother me if it's just a COM import definition.